packages/core/src/config/config.ts

```diff
@@ -263,6 +265,7 @@ export class Config {
   private readonly chatCompression: ChatCompressionSettings | undefined;
   private readonly interactive: boolean;
   private initialized: boolean = false;
+  private readonly logInitialSessionEventEnabled: boolean;
```

Comment:
why is having logInitialSessionEventEnabled needed?

Reply:
I ran into snags in the tests where using a real instance of `Config()` (previously, the tests used mocks). There's some code in `Config()` that was triggering events that I needed to disable.  I wanted to avoid adding an unnecessary `clearEvents` method or otherwise.

Reply:
... Actually, apologies, I just found a mechanism that allowed me to remove this whole adventure. It was not needed.